### PR TITLE
button is fixed for session page in templates

### PR DESCRIPTION
**Pull request by inayahassan23**
Fixes # #5907

#### Changes proposed in this pull request:

-
-
-

#### Checklist

- [ ] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [ ] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ Docker images hosted on [open-event-frontend repository](https://cloud.docker.co
     -   [ember inspector for chrome](https://chrome.google.com/webstore/detail/ember-inspector/bmdblncegkenkacieihfhpjfppoconhi)
     -   [ember inspector for firefox](https://addons.mozilla.org/en-US/firefox/addon/ember-inspector/)
 
-## Contributions Best Practices
+## Best Practices for Contributions
```
Comment (daretobedifferent18) [inline README.md:105]: Whatever changes u made does not relate to anything with the issue so Pls read the issue again carefully and do accordingly

Comment (inayahassan23) [reply README.md]: i remove this

Comment (MeghalBisht) [reply README.md]: it is working fine for me now

File: app/templates/events/view/sessions.hbs [hunk 2]
```diff
@@ -14,7 +14,7 @@
             {{/each}}
           </TabbedNavigation>
         </div>
-        <div class="eight wide right aligned column">
+        <div class="sixteen wide right aligned column">
```
Comment (MeghalBisht) [inline app/templates/events/view/sessions.hbs:17]: Please change this as well

File: app/templates/events/view/speakers.hbs [hunk 3]
```diff
@@ -2,7 +2,7 @@
   <div class="ui grid stackable">
     {{#if (and (not-eq this.session.currentRouteName 'events.view.speakers.edit') (not-eq this.session.currentRouteName 'events.view.speakers.create'))}}
       <div class="row">
-        <div class="six wide column">
+        <div class="{{if this.device.isTablet 'sixteen' 'twelve'}} wide column">
```
Comment (MeghalBisht) [inline app/templates/events/view/speakers.hbs:5]: This is duplicate of #5907, check the comments there

Comment (inayahassan23) [reply app/templates/events/view/speakers.hbs:5]: ok

**Pull request closed without merging**